infra/resources/prod/_modules/custom_roles/cosmos_query.tf

```diff
@@ -0,0 +1,13 @@
+resource "azurerm_role_definition" "cosmos_query" {
+  name        = "CosmosDB Data Operator"
+  scope       = var.resource_group_id
```

Comment:
is that `scope` right? I was expecting to read our cosmos account id instead of `resource_group_id`

Reply:
You are right, the scope has been fixed here [f75052d](https://github.com/pagopa/io-fims/pull/43/commits/f75052dab4023f8dbf6609dda182ff2389a1e4e3)